js/build.ts

```diff
@@ -7,12 +7,20 @@ export interface BuildInfo {
   arch: "x64";
 
   /** The operating system platform. */
-  os: "mac" | "win" | "linux";
+  os: BuildPlatform;
 
   /** The GN build arguments */
   gnArgs: string;
 }
 
+/** The operating system platform. */
+export enum BuildPlatform {
```

Comment:
Rename this to `OperatingSystem`.
And export this from js/deno.ts too. 

CI is failing because of formatting. Run `tools/format.py`

Reply:
Hmm `OperatingSystem` sounds like a really long name, personally preferring `OSType`, `OSKind` (we already have `ErrorKind`) or other similar names...

Reply:
Sure `OSKind` is fine. I think the longness doesn't matter much, since this is a fairly internal bit of code.

Reply:
Agreed with @kevinkassimo . So i rename it to `OSType` ?

Reply:
sure